packages/components/src/templates/next/components/internal/TableOfContents/TableOfContents.tsx

```diff
@@ -1,18 +1,20 @@
-import { BiRightArrowAlt } from "react-icons/bi"
+import { BiMinus } from "react-icons/bi"
 
 import type { TableOfContentsProps } from "~/interfaces"
 
 const TableOfContents = ({ items }: TableOfContentsProps) => {
   return (
-    <div className="flex flex-col gap-3 border-l-4 border-content pl-5">
-      <p className="text-lg font-semibold text-content-strong">On this page</p>
-      <div className="flex flex-col gap-3">
+    <div className="flex flex-col gap-3">
+      <p className="prose-headline-lg-medium text-base-content-strong">
+        On this page
+      </p>
+      <div className="prose-body-base flex flex-col gap-3">
         {items.map(({ anchorLink, content }) => (
           <div className="flex flex-row items-start gap-2">
-            <BiRightArrowAlt className="size-6 shrink-0 text-interaction-main" />
+            <BiMinus className="size-5 shrink-0 fill-base-content-strong" />
```

Comment:
issue: I think the minus sign is not middle aligned
<img width="393" alt="Screenshot 2024-07-29 at 11 18 59" src="https://github.com/user-attachments/assets/214bb4c8-fca5-4ba4-b90c-02931332a661">

Reply:
addressed in https://github.com/opengovsg/isomer/pull/377/commits/26b423430ba1c7aeb875ebcb8c830ef7b79eb434